test/groovy/com/sap/piper/integration/TransportManagementServiceTest.groovy

```diff
@@ -31,43 +31,50 @@ class TransportManagementServiceTest extends BasePiperTest {
 
     @Test
     void retrieveOAuthToken__successfully() {
+        def uaaUrl = 'http://dummy.com/oauth'
```

Comment:
There is a lot of noise here with relocating lines. It is also not needed to use a token that looks like a real token (having `myOAuthToken` as a reference in the test is fully sufficient and maybe even clearer since the name outlines what it is. There is also nothing wrong IMO with defining the response content below inline rather than concatenate variables for that. Having real life string - like the token here - might also be a vector for an attacker since s?he might retrieve use-full information about the structure of a secret.

Basically adding this line 
```
assertThat(loggingRule.log, not(containsString('myOAuthToken')))
```
below does the job (remember the title of this PR: "explicitly check in tests that there is no token in the logs"). With that we have less noise and a better focused PR.

Reply:
Addressed.